resources/js/libraries/extend/inpage_navigation.js

```diff
@@ -0,0 +1,9 @@
+(function (bs, Drupal) {
+  Drupal.behaviors.inpage_navigation = {
+    attach: function attach(context, settings) {
+      new bs.ScrollSpy(document.body, {
+        target: '#bcl-inpage-navigation'
```

Comment:
Same as above, what about we have more than one scrollspy in the same page? Maybe the target must be a class and not an id, we can talk about it with someone else...

Reply:
We need this until BCL 16 is delivered. Then we will change this here and use the class.